components/utilities/filters.js

```diff
@@ -0,0 +1,54 @@
+const Polkadot = "polkadot";
```

Comment:
Moved filters to an independent js module so they can be used by components other than just the RPC